jekyll/_cci2/deployments-and-uploads.md

```diff
@@ -1,8 +0,0 @@
----
```

Comment:
Sure, I can do that, but is it still necessary even if https://circleci.com/docs/2.0/deployments-and-uploads does not exist now?